stdlib/public/core/Sequence.swift

```diff
@@ -605,11 +605,11 @@ public protocol Sequence {
   /// in the same order.
   func _copyToContiguousArray() -> ContiguousArray<Iterator.Element>
 
-  /// Copy a Sequence into an array, returning one past the last
-  /// element initialized.
-  @discardableResult
-  func _copyContents(initializing ptr: UnsafeMutablePointer<Iterator.Element>)
-    -> UnsafeMutablePointer<Iterator.Element>
+  /// Copy `self` into an unsafe buffer, returning a partially-consumed 
+  /// iterator with any elements that didn't fit remaining.
```

Comment:
Please document what it means to return a null iterator?

Can we explicitly state the Sequence._copyContents invariants. e.g. _copyContents will complete the copy, thus return an empty iterator, whenever the buffer argument has sufficient space. Or, more generally, can we say that _copyContents is guaranteed to copy min(count, buffer.count) elements?